plugins/inputs/ravendb/ravendb.go

```diff
@@ -378,52 +375,59 @@ func gatherServer(r *RavenDB, acc telegraf.Accumulator) {
+		"memory_low_memory_severity":                                    mapMemorySeverity(serverResponse.Memory.LowMemorySeverity),
+		"memory_total_swap_size_in_mb":                                  serverResponse.Memory.TotalSwapSizeInMb,
+		"memory_total_swap_usage_in_mb":                                 serverResponse.Memory.TotalSwapUsageInMb,
+		"memory_working_set_swap_usage_in_mb":                           serverResponse.Memory.WorkingSetSwapUsageInMb,
+		"memory_total_dirty_in_mb":                                      serverResponse.Memory.TotalDirtyInMb,
+		"disk_system_store_used_data_file_size_in_mb":                   serverResponse.Disk.SystemStoreUsedDataFileSizeInMb,
+		"disk_system_store_total_data_file_size_in_mb":                  serverResponse.Disk.SystemStoreTotalDataFileSizeInMb,
+		"disk_total_free_space_in_mb":                                   serverResponse.Disk.TotalFreeSpaceInMb,
+		"disk_remaining_storage_space_percentage":                       serverResponse.Disk.RemainingStorageSpacePercentage,
+		"license_type":                                                  serverResponse.License.Type,
+		"license_utilized_cpu_cores":                                    serverResponse.License.UtilizedCpuCores,
+		"license_max_cores":                                             serverResponse.License.MaxCores,
+		"network_tcp_active_connections":                                serverResponse.Network.TcpActiveConnections,
+		"network_concurrent_requests_count":                             serverResponse.Network.ConcurrentRequestsCount,
+		"network_total_requests":                                        serverResponse.Network.TotalRequests,
+		"network_requests_per_sec":                                      serverResponse.Network.RequestsPerSec,
+		"cluster_node_state":                                            mapNodeState(serverResponse.Cluster.NodeState),
+		"cluster_current_term":                                          serverResponse.Cluster.CurrentTerm,
+		"cluster_index":                                                 serverResponse.Cluster.Index,
+		"databases_total_count":                                         serverResponse.Databases.TotalCount,
+		"databases_loaded_count":                                        serverResponse.Databases.LoadedCount,
+		"cpu_machine_io_wait":                                           serverResponse.Cpu.MachineIoWait,
+		"license_expiration_left_in_sec":                                serverResponse.License.ExpirationLeftInSec,
+		"network_last_request_time_in_sec":                              serverResponse.Network.LastRequestTimeInSec,
+		"network_last_authorized_non_cluster_admin_request_time_in_sec": serverResponse.Network.LastAuthorizedNonClusterAdminRequestTimeInSec,
+		"certificate_server_certificate_expiration_left_in_sec":         serverResponse.Certificate.ServerCertificateExpirationLeftInSec,
```

Comment:
Would it make sense to sort this alphabetically in case someone is looking for a certain field?

Reply:
sorted

Reply:
Perfect, thx! Please mark as resolved.